src/cmd/api/goapi.go

```diff
@@ -80,6 +80,7 @@ var contexts = []*build.Context{
 	{GOOS: "openbsd", GOARCH: "386"},
 	{GOOS: "openbsd", GOARCH: "amd64", CgoEnabled: true},
 	{GOOS: "openbsd", GOARCH: "amd64"},
+	{GOOS: "windows", GOARCH: "arm64"},
```

Comment:
Looks like the tuples tend to be grouped by OS, we should follow that convention. Also, is there a reason Windows + arm isn't here?

Reply:
Actually, I am not convinced this is even needed. Removing it from the pr.